src/LayoutManager/LayoutManager.ts

```diff
@@ -130,11 +142,27 @@ export class LayoutManager {
     const { target } = context;
     const { canvas } = target;
     // handle layout triggers subscription
+    // @TODO: gate the registration when the group is interactive
```

Comment:
This is redundant IMO since if the group is not interactive nothing will trigger imperative layout because the user will not be able to select and transform a nested object.
It does subscribe the objects with no need but if the group becomes interactive we save the dev from a gotcha needing to subscribe the targets when toggling interactivity

Reply:
> As a counter argument interactive group is an edge case, for now we don't need to go to the extreme that objects get registered for handlers only when they get selected or when a trasform action starts, but at least when the group is interactive yes.

This is not bad

Reply:
However toggling interactivity... I am not sure. There is another PR I did which I mentioned in this PR regarding it
I will find it